test/src/main/java/org/apache/accumulo/test/TransportCachingIT.java

```diff
@@ -158,7 +124,7 @@ public void testCachedTransport() throws InterruptedException {
       while (fifth == null) {
         try {
           // Get a cached transport
-          fifth = pool.getAnyTransport(servers, true).getSecond();
+          fifth = pool.getAnyTransport(servers.subList(0, 1), true).getSecond();
```

Comment:
Do we want this to loop indefinitely? If this is null, could it ever become non-null?

Reply:
The looping seemed kinda sketchy to me, but I didn't look into it in detail.

Reply:
If on the first attempt it gets a TTransportException and on the 2nd attempt it succeeds then it seems like it could become non-null, right?  So maybe the looping is not so sketchy after all.

Reply:
I'm not really sure. I was confused by what the test was trying to do, and the possible outcomes it was trying to account for with the loop, since none of that is documented :smiley_cat:
I will defer to you.